node/src/config_files/mod.rs

```diff
@@ -137,6 +137,7 @@ fn chainstate_config(
 fn p2p_config(config: P2pConfigFile, options: &RunOptions) -> P2pConfigFile {
     let P2pConfigFile {
         bind_address,
+        add_node,
```

Comment:
There's an issue that has been discussed earlier but has been omitted for a while and we have to come back to it sooner or later. The issue is that not all peers are equal. There are peers that we want to persistently have, and others that we want to just give a "shot" at connecting at. I think in Substrate they call the former "reserved" peers or something. We should also consider doing that. This is important because some times you have peers with whom you want to maintain a connection no matter what. I don't know if you're gonna do that in this PR, but please put that into consideration, whether in an issue or this PR. Please research this topic when you can.

Reply:
Right, we will need this too. But let's please do it later.

Reply:
Please open an issue for it then resolve this comment.